src/page-modules/departures/server/journey-planner/index.ts

```diff
@@ -191,3 +206,19 @@ type RecursivePartial<T> = {
     ? RecursivePartial<T[P]>
     : T[P];
 };
+
+const filterTransportModes = (
```

Comment:
These functions could possibly be moved to the transport-mode utils? Or maybe that folder is more for the component 🤷

Reply:
I think we should separate this from the client component to avoid misusing the graphql type in the client. However, it might be reasonable to add a utils folder for functions like this? 🤔